pom.xml

```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>2.6.1</version>
+      <version>3.11.4</version>
```

Comment:
Why the protobuf change is not in master version?

Reply:
Weird, we do have it in v1.x master: https://github.com/awslabs/amazon-kinesis-client/commit/dd72cf96e5436b0002a5ebf363027d86e46f11b3

Reply:
Not sure why it contains this diff. But double checked that we are merging to the KCL 1.x public branch. And this change does exist in v1.x master already